### PR TITLE
Functional tests - Refacto Advanced Parameters tests

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Refacto Advanced Parameters tests
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no
| How to test?      | no need
| Possible impacts? | no

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/26214)

### CODE REVIEW

**Review by boubkerbribri — DISMISSED** [review]

File: tests/UI/campaigns/functional/BO/14_advancedParameters/06_team/profiles/05_paginationAndBulkActions.js [hunk 1]
```diff
@@ -15,18 +18,15 @@ const employeesPage = require('@pages/BO/advancedParameters/team/index');
 const profilesPage = require('@pages/BO/advancedParameters/team/profiles/index');
 const addProfilePage = require('@pages/BO/advancedParameters/team/profiles/add');
 
-// Test context imports
-const testContext = require('@utils/testContext');
-
-const baseContext = 'functional_BO_advancedParameters_team_profiles_paginationProfiles';
+const baseContext = 'functional_BO_advancedParameters_team_profiles_paginationAndBulkActionsProfiles';
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/BO/14_advancedParameters/06_team/profiles/05_paginationAndBulkActions.js]:
```suggestion
const baseContext = 'functional_BO_advancedParameters_team_profiles_paginationAndBulkActions';
```

**Review by Progi1984 — COMMENTED** [review]
@nesrineabdmouleh A small feedback

File: tests/UI/campaigns/functional/BO/14_advancedParameters/06_team/profiles/05_paginationAndBulkActions.js (hunk 1 above)
Comment (nesrineabdmouleh) [reply tests/UI/campaigns/functional/BO/14_advancedParameters/06_team/profiles/05_paginationAndBulkActions.js]: Done thx :+1:

**Review by NeOMakinG — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Review by boubkerbribri — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @nesrineabdmouleh & @boubkerbribri

**Pull request merged**